services/src/main/java/org/exoplatform/chat/service/DocumentService.java

```diff
@@ -188,6 +189,8 @@ public Response persistFile(@Context SecurityContext securityContext,
     String workspace = node.getSession().getWorkspace().getName();
     String repository = ((ManageableRepository) node.getSession().getRepository()).getConfiguration().getName();
     String nodePathWithWorkspace = workspace + node.getPath();
+    String fileNameFromPath = node.getPath().substring(node.getPath().lastIndexOf("/") + 1);
```

Comment:
Why don't we use node.getName() or if the node title exists, node.getProperty("exo:title").getString() ?

Reply:
done !